python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -1006,39 +966,6 @@ def _parse_graph_initializers(self, graph: onnx.onnx_ml_pb2.GraphProto):
             array = self._parse_array(init_tensor)
             self._nodes[init_tensor.name] = relax.const(array)
 
-    def _sanitize_name(self, name: str) -> str:
```

Comment:
Is there a particular reason for removing this?

Reply:
No, I think this is a rebase issue. Would you please send a PR to add this back?

Reply:
Sorry, but I do not think I can do that. If this is a rebase issue, then I think the way to go is to do a correct rebase instead. I can also see that the previous tests have been removed, including the `check_correctness` function.